sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/signaling/TrouterUtils.java

```diff
@@ -0,0 +1,343 @@
+package com.azure.android.communication.chat.signaling;
+
+import com.azure.android.communication.chat.signaling.properties.ChatEventId;
+import com.microsoft.trouterclient.ITrouterResponse;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+public class TrouterUtils {
```

Comment:
If this is a type only used internally and should not be exposed to the end user, then consider making it package private or move to implementation package.